CONTRIBUTING.md

```diff
@@ -51,47 +51,21 @@ npm install -g bower
 
 # Install Bower dependencies
 bower install
-
-# Create a .env file and populate it with the necessary API keys and secrets:
-touch .env
 ```
-
-Edit your `.env` file with the following API keys accordingly. If you only use email login, only the `MONGOHQ_URL`, `SESSION_SECRET`, `MANDRILL_USER` and `MANDRILL_PASSWORD` fields are necessary. Keep in mind if you want to use more services you'll have to get your own API keys for those services. If you only use a subset or no OAuth2 authentication methods, you may want to remove them from ```server/passport-providers.js``` - otherwise the server will complain about missing clientIDs at launch.
-
+*Private Environment Variables (API Keys)*
+```bash
+# Create a copy of the "sample.env" and name it as ".env".
+# Populate it with the necessary API keys and secrets:
+cp sample.env .env
 ```
-MONGOHQ_URL='mongodb://localhost:27017/freecodecamp'
-
-FACEBOOK_ID=stuff
-FACEBOOK_SECRET=stuff
-
-GITHUB_ID=stuff
-GITHUB_SECRET=stuff
 
-GOOGLE_ID=stuff
-GOOGLE_SECRET=stuff
+Edit your `.env` file and modify the API keys only for services that you will use.
 
-LINKEDIN_ID=stuff
-LINKEDIN_SECRET=stuff
+Note : Not all keys are required, to run the app locally, however `MONGOHQ_URL` is the most important one.
 
-MANDRILL_PASSWORD=stuff
-MANDRILL_USER=stuff
+If you only use email login, only the `MONGOHQ_URL`, `SESSION_SECRET`, `MANDRILL_USER` and `MANDRILL_PASSWORD` fields are necessary.
```

Comment:
You can login but it will throw an error in while you sign up:

![3](https://cloud.githubusercontent.com/assets/1884376/13194944/274f7fd6-d7c9-11e5-82f3-3736240f8c52.PNG)

I guess that's the only time we are sending an email to the user. Maybe we can say:
> setting the `MANDRILL_USER` and `MANDRILL_PASSWORD` is not necessary but it will throw an exception when you sign up. This may be ignored, you will still be able to login.

I guess a contributor will need this only in a case when he/she is working on AUTH related stuff?